src/carriers/zfp_portmonitor/cmake/FindZFP.cmake

```diff
@@ -0,0 +1,37 @@
+#
```

Comment:
Duplicated with `cmake/ycm-0.2/find-modules/FindZFP.cmake` , I think this can be removed?